examples/resources/kentik-synthetic_test-bulk-create/agents_filtered_by_distance/resource.tf

```diff
@@ -0,0 +1,85 @@
+// Create a test with list of agents that are located within given radius from coordinates.
+// Inputs: latitude, longitude, radius.
+
+data "kentik-synthetics_agents" "agents" {
+  latitude  = 50.55
+  longitude = 5.5
+  distance  = 9000
+}
+
+locals {
+  agents_ids = [for agent in data.kentik-synthetics_agents.agents.items: agent.id]
+}
+
+resource "kentik-synthetics_test" "private-agents-test" {
+  name      = "private-agents-test"
+  type      = "hostname"
+  device_id = "75702"
+  status    = "TEST_STATUS_ACTIVE"
+  settings {
+    hostname {
+      target = "www.example.com"
+    }
+    agent_ids = local.agents_ids
+    period = 61
+    count  = 2
+    expiry = 5001
+    limit  = 4
+    tasks = tolist([
+      "ping",
+      "traceroute"
+    ])
+    health_settings {
+      latency_critical      = 1
+      latency_warning       = 2
+      packet_loss_critical  = 3
+      packet_loss_warning   = 4
+      jitter_critical       = 5
+      jitter_warning        = 6
+      http_latency_critical = 7
```

Comment:
The comment above applies to all resource files used in examples. Let's minimize them, so that they can be used as good starting point for users creating their own.

Reply:
The `health_settings` are `optionalOnCreate` and the empty list cannot be passed. I completely removed them from examples, is that ok?

Reply:
Yes.